src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {' Posted by  '}
+        <UserInfo user={post.user} />
```

Comment:
user can be null, so better to add the check here
<img width="576" alt="image" src="https://github.com/user-attachments/assets/b3c86f1d-9f4d-49ca-9264-d8142ab068a3">
```suggestion
        {post.user && <UserInfo user={post.user} />}
```